theano/gpuarray/dnn.py

```diff
@@ -361,7 +361,7 @@ def do_constant_folding(self, node):
         return False
 
     def __init__(self, border_mode, subsample=(1, 1), conv_mode='conv',
-                 precision="float32"):
+                 precision="float32", precision="float64"):
```

Comment:
This will fail. You have 2 time the same keyword precision. Can you fix that rapidly?

Also, why do you want to use float64 on GPU? Do you know that most GPU have float64 very slow? Mostly out of curiosity and to stay up to date on people usage of Theano.